tests/run-make/raw-dylib-import-name-type/rmake.rs

```diff
@@ -0,0 +1,36 @@
+// `raw-dylib` is a Windows-specific attribute which emits idata sections for the items in the
+// attached extern block,
+// so they may be linked against without linking against an import library.
+// To learn more, read https://github.com/rust-lang/rfcs/blob/master/text/2627-raw-dylib-kind.md
+// This test uses this feature alongside `import_name_type`, which allows for customization
+// of how Windows symbols will be named. The correctness of this feature is checked by comparison
+// with expected output.
```

Comment:
Suggestion: correctness -> smoke test, since I don't think we can *prove* the feature is correct, AFAICT this is more like a sanity check.